src/context/UserContext.tsx

```diff
@@ -1,34 +1,32 @@
-import React, { ReactNode, createContext, useContext, useState } from 'react';
-import { PersonStructure } from '../model/Person';
+import React, {
+  PropsWithChildren,
+  createContext,
+  useContext,
+  useState,
+} from 'react';
+import { PersonStructure } from '../types/Person';
+
+const initialUser: PersonStructure = {
```

Comment:
Ten naming to bude chtít ještě dostat pod kůži, občas mi není úplně jasné, co to bude. Ono je to ale o zvyku. U tvých pojmenování mi většinou přijdou moc krátké, možná by pomohlo je mít specifičtější (tj. numberOfFollows, canBeFollowed, isFollowable, ...). Důležité je naznačit, kdy se jedná o sloveso (díky can, is, are, on, handle, ...) a radši delší a jasnější než kratší, ale tak, že za měsíc nebudeš vědět, co jsi tím chtěla říct.

Např. když vidím follow, nevím, jestli to může znamenat, že já někoho followuju, někdo followuje mě, někdo je schopný kliknout na follow nebo je zablokovaný a proto follow nikdy nikomu dát nemůže nebo je to počet follows, které ten uživatel má nebo počet follows, které někomu dal, ...